proftpd/sftp.yaml

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
The "_mapdata" exporter shows that there is a difference between how this file is used in comparison to others.

Extract of /tmp/salt_mapdata_dump.yaml 

```
---
values:
  AllowOverwrite: '"on"'
  AllowRetrieveRestart: '"on"'
  AllowStoreRestart: '"on"'
[...]
  Modules:
    mod_ban: true
    mod_copy: true
    mod_ctrls_admin: true
 [...]
  proftpd:
    SFTP:
      AllowOverwrite: 'on'
      DefaultRoot: '~'
      DeleteAbortedStores: 'on'
```
Keys are added under a `proftpd` key, unlike others

Reply:
I merge default.yaml and sftp.yaml, it resolve this issue